weave-js/src/core/_external/util/urls.ts

```diff
@@ -53,3 +71,10 @@ export function makeNameAndID(id: string, name?: string): string {
   }
   return name != null ? `${encodeURIComponent(name)}--${id}` : id;
 }
+
+export function reportViewWithoutPublished(r: ReportParams) {
+  return `/${r.entityName}/${r.projectName}/reports/${makeNameAndID(
+    r.reportID,
+    r.reportName
+  )}`;
+}
```

Comment:
Makes me sad we'll have duplicate report url functions in weave and core, but i understand why :(

Reply:
Indeed 🥲  It's gonna be even worse when you see what I have to copy over for the "new report" logic 😬